python-avd/pyavd/_eos_cli_config_gen/j2templates/eos/router-bgp.j2

```diff
@@ -971,6 +974,22 @@ router bgp {{ router_bgp.as }}
 {%         if router_bgp.address_family_evpn.route.import_overlay_index_gateway is arista.avd.defined(true) %}
       route import overlay-index gateway
 {%         endif %}
+{%         if router_bgp.address_family_evpn.evpn_ethernet_segment is arista.avd.defined %}
+{%             set domain_list = ['all', 'local','remote'] %}
```

Comment:
You have already restricted the values of domain in schema. So no need of this domain_list.

Reply:
This is more to ensure that it renders in the correct order